build-conflicts.php

```diff
@@ -191,14 +193,22 @@ static function (array $components): array {
     $cloneAdvisories();
     $cloneRoaveAdvisories();
 
-    $getAdvisories = (new GetAdvisoriesFromMultipleSources(
+    // sources:
+    $getAdvisoriesSources = (new GetAdvisoriesFromMultipleSources(
         (new GetAdvisoriesFromFriendsOfPhp($buildDir . '/security-advisories')),
         (new GetAdvisoriesFromGithubApi(
             new Client(),
             $token,
         )),
     ));
 
+    $overwriteRuleProvider = (new RuleProviderFactory())();
```

Comment:
IMO a file with `list<callable(Advisory): Advisory>` would be simpler here - those callables can be turned into testable classes too